### PR TITLE
Add Percona support for Debian

### PR DESCRIPTION
**Pull request by clauded**
Percona's repos support Ubuntu and Debian. This PR adds support for the Debian distribution using Percona. This has been successfully tested with Jessie (8.3)

### CODE REVIEW

File: manifests/repo.pp [hunk 1]
```diff
@@ -15,16 +15,22 @@
   $apt_percona_repo_key_server = 'keys.gnupg.net',
   $apt_percona_repo_include_src = false,
 
-  # Ubuntu/mariadb
-  $apt_mariadb_repo_location = 'http://mirror.aarnet.edu.au/pub/MariaDB/repo/5.5/ubuntu',
+  # Ubuntu-Debian/mariadb
+  $apt_mariadb_repo_location = $::opratingsystem ? {
```
Comment (michaeltchapman) [inline manifests/repo.pp]: typo on operatingsystem

**Comment by michaeltchapman** [comment]
Hey, thanks for looking at this, could you please squash the commits down to 1?

**Comment by clauded** [comment]
Commits have been squashed.

**Comment by michaeltchapman** [comment]
Thanks, there's also a typo 'opratingsystem', other than that it looks good.

**Comment by clauded** [comment]
Typo corrected. Any chance the code being commited?

**Comment by clauded** [comment]
Should be ok now.

**Comment by michaeltchapman** [comment]
Cheers

**Pull request merged**